content/collective_intro.tex

```diff
@@ -1,11 +1,55 @@
-\emph{Collective routines} are defined as communication or synchronization
-operations on a group of \acp{PE} called an active set. The collective
-routines require all \acp{PE} in the active set to simultaneously call the
-routine.  A \ac{PE} that is not in the active set calling the collective
-routine results in undefined behavior.  All collective routines have an
+\emph{Collective routines} are defined as \newtext{coordinated} communication or
+synchronization operations performed \newtext{by} \oldtext{on} a group of \acp{PE} \oldtext{called an active set}.
+{\color{ForestGreen} % <start-newtext>
+\openshmem provides collective routines that operate on teams
+(see Section~\ref{subsec:team} on team management)
+\DeprecateInline{and active sets}.
+
+The team-based collective routines are performed with respect to a valid
+\openshmem team, which is specified by a team handle argument.
+Team-based collective operations require all \acp{PE} in the team to call
+the routine in order for the operation to complete.
+
+A team-based collective operation is considered \emph{active} on a \ac{PE}
+when that \ac{PE} calls one of the team-based collective routines.
+Such operations may be blocking or nonblocking and the nature of which
+determines the conditions under which the operation is considered complete:
```

Comment:
Nonblocking collectives discussion would be easier to add as a separate PR from this one, since the notion of nonblocking collectives is still under discussion.